fxa/core.py

```diff
@@ -278,12 +285,7 @@ def get_email_status(self):
         return resp
 
     def verify_email_code(self, code):
-        body = {
-            "uid": self.uid,
-            "code": code,
-        }
-        url = "/recovery_email/verify_code"
-        self.apiclient.post(url, body)  # note: not authenticated
+        return self.client.verify_email_code(self.uid, code) # note: not authenticated
```

Comment:
>We can squash your commit if you want before merging them into master but I actually like to show that software is not perfect at first and that it improves with time

I like that approach! So if it doesn't bother you, you need not squash it for me.